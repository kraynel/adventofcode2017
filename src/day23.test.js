const { executeProgram, executeSmart } = require("./day23");
const fs = require("fs");

describe("executeProgram", () => {
  it("should get the debug value", done => {
    fs.readFile("inputs/day23.txt", "utf8", (err, data) => {
      expect(executeProgram(data)).toMatchSnapshot();
      console.log(executeSmart());
      done();
    });
  });

  // it("should get the non debug value", done => {
  //   fs.readFile("inputs/day23.txt", "utf8", (err, data) => {
  //     expect(executeProgram(data, false)).toMatchSnapshot();
  //     done();
  //   });
  // });
});
