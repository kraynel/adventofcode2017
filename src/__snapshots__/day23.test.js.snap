// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`executeProgram should get the debug value 1`] = `3025`;
